Create complexity_measure.py

EthanGuo666 opened the pull request:
Add the function which measures the code complexity.

Commit:
Create complexity_measure.py

add a function which measures the code complexity.

Commit:
Update complexity_measure.py

Add measurement of bit operator and modulo operator

hehao98 commented:
For easy reuse of the complexity measure function, I suggest the function should return its result.

Also, for testing, you can use the following:
```python
if __name__ == '__main__':
    # Testing code for the function here
```

Commit:
Update complexity_measure.py

Modify the code according to Hehao's suggestion, and change an inconsistent variable name

EthanGuo666 commented:
Very pleased to receive feedback from code reviewers, now complexity_measure.py is modified according to He hao's suggestion.
